serverless.yml

```diff
@@ -51,6 +57,11 @@ resources:
           Ref: WebAppS3Bucket
         PolicyDocument:
           Statement:
+            - Sid: "1"
```

Comment:
Sid is an optional identifier you can use to keep track of what a policy does. Setting it to "1" says nothing. You should set it to something that describes the policy.

Cf. https://stackoverflow.com/questions/47514945/what-is-sid-attribute-use-for-in-key-policies

Reply:
Fixed. I set Sid to a string that describes the policy.